.github/workflows/wc-pana-evaluation.yml

```
@@ -0,0 +1,120 @@
+    permissions:
+      pull-requests: write
+    env:
+      BASE_DIR: /tmp/pana-evaluation
+      PANA_OUTPUT_FILE: /tmp/pana-evaluation/pana_output.json
+      COMMENT_FILE: /tmp/pana-evaluation/comment.md
+      PACKAGE_DIR: ${{ inputs.package_dir }}
+      PACKAGE_NAME: ${{ inputs.package_dir == 'packages/flutter_whisper_kit_apple' && 'Flutter Whisper Kit Apple' || 'Flutter Whisper Kit' }}
+    
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)